go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/google/flatbuffers
```

Comment:
Is this required?

Reply:
yes. 
go module support in 1.11+. 
go.mod is key of go package management.